### DISCUSSION TITLE
How do you build this?

### DISCUSSION THREAD

**Discussion by dopadream** [MoreMcmeta/core#31] [Core Dev Questions] [answered]
I'm trying to build the emissive plugin and I'm getting an error saying I need a username and password for the maven repository.

**Comment by soir20** [comment, marked as answer]
You'll need to follow the [Maven instructions for GitHub packages](https://github.com/MoreMcmeta/core/wiki/Plugin-Docs:-Maven#github-packages) in the plugin docs. (I'll add a link to this to both READMEs; the core one is missing it and the emissive plugin README is a bit light at the moment.)

**Reply by dopadream** [reply]
Thank you!
